alibi_detect/saving/loading.py

```diff
@@ -262,24 +264,24 @@ def _load_model_config(cfg: dict,
         raise FileNotFoundError("The `src` field is not a recognised directory. It should be a directory containing "
                                 "a compatible model.")
 
-    if backend == 'tensorflow':
+    if flavour == 'tensorflow':
         model = load_model_tf(src, load_dir='.', custom_objects=custom_obj, layer=layer)
+    elif flavour == 'sklearn':
+        model = load_model_sk(src, load_dir='.')
     else:
-        raise NotImplementedError('Loading of non-tensorflow models not currently supported')
+        raise NotImplementedError('Loading of PyTorch models not currently supported')
 
     return model
 
 
-def _load_embedding_config(cfg: dict, backend: str) -> Callable:  # TODO: Could type return more tightly
+def _load_embedding_config(cfg: dict) -> Callable:  # TODO: Could type return more tightly
```

Comment:
No longer need to be passed `backend` as `flavour` is contained in the `EmbeddingConfig`. 

The ultimate goal here is to make these artefact config's self-contained. This will pave the way towards a more OOP approach! (@mauicv )